Gruntfile.js

```diff
@@ -16,13 +16,55 @@ module.exports = function(grunt) {
     },
     browserify: {
       standalone: {
-        src: [ 'src/<%= pkg.name %>.js' ],
+        src: 'src/<%= pkg.name %>.js',
         dest: 'dist/<%= pkg.name %>.js',
         options: {
           browserifyOptions: {
-            standalone: 'ContextParserHandlebars'
+            standalone: 'Handlebars'
```

Comment:
it will pollute the name of original Handlebars, should we use "SecureHandlebars"?